### ISSUE TITLE
We were unable to create an instance of the requested type UnityEditor.XR.OpenXR.OpenXRPackageSettings

### ISSUE THREAD

**Issue by jonkelling** [KhronosGroup/OpenXR-Docs#139]
I updated from Unity 2022.2.1f1 to 2022.2.2f1 which updated OpenXR from 1.5.3 to 1.6.0. Upon opening the project, or rebuilding assemblies, I get 2 errors about OpenXRPackageSettings. One is "We were unable to create an instance of the requested type UnityEditor.XR.OpenXR.OpenXRPackageSettings." I noticed it a new OpenXRPackageSettings asset file was created. I tried removing the old one, deleting both (to recreate it) and the error persists. There are no other compilation errors, however when I try to deploy to an Android device, I get 3 errors from com.unity.services about not being able to find Newtonsoft.Json—obviously not OpenXR, and I'm not sure how it could be related, but mentioning it just in case. I'll come back to edit this with the error details when I get a chance.

**Comment by rpavlik** [comment]
This is probably better to post in the unity forums or maybe the OpenXR forums: it's a unity specific problem that does not have anything to do with the OpenXR standard. Good luck!

**Comment by jonkelling** [comment]
I posted in the Unity forums. https://forum.unity.com/threads/the-type-or-namespace-name-newtonsoft-could-not-be-found-and-more-in-2022-2-2f1-update.1386573/

In brief: I tried creating a new blank URP project in Unity 2022.2.2f1, added OpenXR 1.5.3—no problems. When updating to OpenXR 1.6.0, all of the errors I mentioned came up right away. I can try again in a different version of Unity when I have the chance, and just see where the troubleshooting goes in the Unity forums for now.

Thanks again.
